src/python/WMComponent/DBS3Buffer/DBSUploadPoller.py

```diff
@@ -109,6 +113,10 @@ def uploadWorker(workInput, results, dbsUrl, gzipEncoding=False):
                 logging.exception(msg)
```

Comment:
Two lines above, we call `parseDBSException`. I would suggest to completely remove it - and remove the function as well - given that we now have an specific class for parsing the DBS server error.

Reply:
Please elaborate more as I do not understand from your message what should be removed.

Reply:
This line:
```
                reason = parseDBSException(exString)
```

and the function implementation, give that it will no longer be used.

Reply:
ok, now it is clear, I removed those lines.